releasenotes/notes/callback-for-NN-classifier-regressor-8e1d5c887335e6d9.yaml

```
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    The class ``TrainableModel``, and hence its subclasses ``NeuralNetworkClassifier``, ``NeuralNetworkRegressor``, ``VQR``, 
+    ``VQC``, have a new optional argument ``callback``. It is a reference to a user's callback function that can access 
+    the intermediate training data to track the optimization process. The default value is ``Nonee`` and returns ``None``.
```

```suggestion
    the intermediate training data to track the optimization process. The default value is ``None`` and returns ``None``.
```
I find the latter sentence confusing as it talks about the callback parameter value, which being optional defaults to None, and the return part I guess is about the callback itself and its return value which I think for the release note can just be dripped. Even the prior sentence could simply say A user can optionally supply a callback for access to the intermediate training data... such that being explicit about None in the release note is just not needed